cluster/manifests/deletions.yaml

```diff
@@ -31,6 +31,14 @@ pre_apply:
     application: stackset-controller
   namespace: kube-system
   kind: Deployment
+- labels:
+    application: kubernetes-dashboard
+  namespace: kube-system
+  kind: Deployment
+  - labels:
```

Comment:
This is indented too much which makes CLM fail in the e2e create cluster step.

Reply:
Wops, thanks!